packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -28,401 +28,57 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/REA
-+          properties:
-+            match:
-+              description: Configuration for namespace exclusion
-+              items:
-+                properties:
-+                  excludedNamespaces:
-+                    items:
-+                      type: string
-+                    type: array
-+                  processes:
-+                    items:
-+                      type: string
-+                    type: array
-+                type: object
-+              type: array
-+            readiness:
-+              description: Configuration for readiness tracker
-+              properties:
-+                statsEnabled:
-+                  type: boolean
-+              type: object
-+            sync:
-+              description: Configuration for syncing k8s objects
-+              properties:
-+                syncOnly:
-+                  description: If non-empty, only entries on this list will be replicated into OPA
-+                  items:
-+                    properties:
-+                      group:
-+                        type: string
-+                      kind:
-+                        type: string
-+                      version:
-+                        type: string
-+                    type: object
-+                  type: array
-+              type: object
-+            validation:
-+              description: Configuration for validation
-+              properties:
-+                traces:
-+                  description: List of requests to trace. Both "user" and "kinds" must be specified
-+                  items:
-+                    properties:
-+                      dump:
-+                        description: Also dump the state of OPA with the trace. Set to `All` to dump everything.
-+                        type: string
-+                      kind:
-+                        description: Only trace requests of the following GroupVersionKind
-+                        properties:
-+                          group:
-+                            type: string
-+                          kind:
-+                            type: string
-+                          version:
-+                            type: string
-+                        type: object
-+                      user:
-+                        description: Only trace requests from the specified user
-+                        type: string
-+                    type: object
-+                  type: array
-+              type: object
-+          type: object
-+        status:
-+          description: ConfigStatus defines the observed state of Config
-+          type: object
-+      type: object
-+  version: v1alpha1
-+  versions:
-+  - name: v1alpha1
-+    served: true
-+    storage: true
+@@ -3,7 +3,6 @@
+ metadata:
+   annotations:
+     controller-gen.kubebuilder.io/version: v0.3.0
+-    helm.sh/hook: crd-install
+     helm.sh/hook-delete-policy: before-hook-creation
```

Comment:
If this is no longer a Helm hook, we should remove the `hook-delete-policy` annotation as well.

Applies to all the CRDs on this chart

Reply:
Very weird thing anyway. This would attempt to delete the CRD on re-install or something ? Yea let's remove them to be safe.

Reply:
Wait if the CRD's were moved to the CRD folder, what file is this modifying ?

Reply:
@cbron This modifies their crds in the `crds` directory of their upstream. Their crds used to be in `template` because it was a helm 2 chart and we had to do this change and also move them to a `crds` directory.

Reply:
re: https://github.com/rancher/charts/pull/670#discussion_r491739018 I don't think this would do anything since it's not defined as a Helm hook resource. Most likely would just get ignored afaik unless Helm does something super weird

Reply:
@aiyengar2 I know that `crd-install` hook gets ignored by Helm 3 (not sure about others), but it doesn't hurt to remove them for peace of mind.

Reply:
@PennyScissors we definitely should remove them, but what I meant with https://github.com/rancher/charts/pull/670#discussion_r491752108 is that because we are removing the `helm.sh/hook` annotation from the pod, `helm.sh/hook-delete-policy` likely won't be evaluated either. So we might as well remove both.

Reply:
I will remove them in the next commit, also remove old tgz.